css/_variables.scss

```diff
@@ -89,7 +89,7 @@ $sidebarWidth: 200px;
  */
 $tooltipsZ: 901;
 $toolbarZ: 900;
-$overlayZ: 800;
```

Comment:
Addressed in https://github.com/jitsi/jitsi-meet/commit/98de4c90b543b95d5ba5e481e4180f6d32462528